recipes/circle-fit/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "circle-fit" %}
+{% set version = "0.1.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: d4173696cca074c9e5a2791169854064730b0fa53836326c55634d947222d794
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - matplotlib
+    - numpy
+    - pip
+    - python
+    - scipy
+  run:
+    - matplotlib
+    - numpy
+    - python
+    - scipy
+
+test:
+  imports:
+    - circle_fit
+
+about:
+  home: "https://github.com/AlliedToasters/circle-fit"
+  license: MIT
+  license_file: LICENSE.txt
```

Comment:
Add a comment above the `licence_file` line that includes a link to the following issue:

https://github.com/AlliedToasters/circle-fit/issues/7

as a reminder to look for the license file in the source distribution in future releases so the manually packaged one can be removed.